BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -10,29 +10,38 @@ import Foundation
 import UIKit
 import SWRevealController
 
-protocol SideMenuFilterViewControllerDelegate {
-    
-    func onButtonTap(sender: UIButton)
-    
-    /*
-   var radius = distanceFilter
-   SearchStores.searchstoreslocations(userLocation) {
-     https: apicall.call/\(radius)
-     }
- */
-    
-}
 
 class SideMenuFilterViewController: UITableViewController, SWRevealViewControllerDelegate {
     
     @IBOutlet weak var slider: UISlider!
     @IBOutlet weak var distanceLabel: UILabel!
-    
     @IBOutlet weak var openNowLabel: UILabel!
-    
     @IBOutlet weak var openNowSwitch: UISwitch!
+    @IBOutlet weak var restaurantButton: UIButton!
+    @IBOutlet weak var storesButton: UIButton!
+    @IBOutlet weak var applyButton: UIButton!
+    @IBOutlet weak var cancelButton: UIButton!
+    @IBOutlet weak var contentViewCell: UIView!
```

Comment:
What does this represent? 
The name suggests it is a table view cell, and you should never have an outlet to a cell.